vars/kola.groovy

```diff
@@ -146,30 +146,10 @@ def call(params = [:]) {
             def id = marker == "" ? "kola-upgrade" : "kola-upgrade-${marker}"
             ids += id
             try {
-                runKola(id, "--upgrades")
+                runKola(id, 'run-upgrade',  "--upgrades")
             } catch(e) {
                 // If we didn't even get logs then let's remove them from the list
                 if (shwrapRc("cd ${cosaDir} && cosa shell -- test -d ${outputDir}/${id}") != 0) {
                     ids.remove(id)
                 }
                 if (params["allowUpgradeFail"]) {
-                    warn(e.getMessage())
```

Comment:
Is this deleted code expected?

Reply:
nope - looks like I messed up my rebase when I consolidated commits. Thanks for catching!